### DISCUSSION TITLE
Broken the VSCode theme switching

### DISCUSSION THREAD

**Discussion by CDTay** [prasanthrangan/hyprdots#120] [Q&A] [answered]
When I opened VSCode it said that three of the themes needed updating, so I clicked update and then clicked the "reload required" button on them like an idiot. Now VSCode doesn't automatically switch theme with the rest of the desktop on one specifically, it just stays as the previous desktops theme (previous desktop being the cyberpunk one).

Seems like updating the themes then reloading broke a file somewhere, any ideas what one?

**Comment by prasanthrangan** [comment, marked as answer]
Hi, updating the vscode theme will break it as i had modified the color codes and names of the original themes...
I'll try to create an extension for vscode will all my themes to avoid this in future.

So for now, you either change/apply the themes manually or remove the updated extensions and restore the old version from [here](https://github.com/prasanthrangan/hyprdots/blob/main/Source/arcs/Code_Extensions.tar.gz)

**Reply by CDTay** [reply]
Thanks so much, that worked perfectly
